plugins/CAS_SD/H_apply.irp.f

```diff
@@ -20,18 +20,22 @@
 
 
 s = H_apply("CAS_S",do_double_exc=False)
```

Comment:
`do_double_exc=False` argument is not enough? We have to put  `s.unset_double_excitations()` ?